LowLevelDesign.md

```diff
@@ -1,5 +1,8 @@
 # Low Level Design
 
+## 55+ LLD Solutions
```

Comment:
please dont add this as the top item in the file. Basic concepts should be at the top. Add this after the questions.

Reply:
Done, please check